From f85b13ec5fd038ae88f27c4221fe82e9ebdbe96e Mon Sep 17 00:00:00 2001
From: Gabriele M <moto.falcon.git@gmail.com>
Date: Sun, 20 Nov 2016 22:04:36 +0100
Subject: [PATCH] losetup: Fix loop mounting

Vold creates the loop blocks in /dev/block.

Change-Id: Ie926ce672deae4a3494aad2ea19d44d7be60ec97

Conflicts:
	toys/other/losetup.c
---
 toys/other/losetup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/toys/other/losetup.c b/toys/other/losetup.c
index e8f02aa9..ea30d91e 100644
--- a/toys/other/losetup.c
+++ b/toys/other/losetup.c
@@ -66,7 +66,7 @@ static void loopback_setup(char *device, char *file)
     // mount -o loop depends on found device being at the start of toybuf.
     if (cfd != -1) {
       if (0 <= (i = ioctl(cfd, 0x4C82))) { // LOOP_CTL_GET_FREE
-        sprintf(device = toybuf, "/dev/loop%d", i);
+        sprintf(device = toybuf, "/dev/block/loop%d", i);
         // Fallback for Android
         if (access(toybuf, F_OK)) sprintf(toybuf, "/dev/block/loop%d", i);
       }
-- 
2.17.1

